### ISSUE TITLE
mixin local RPM adding broken since last release

### ISSUE THREAD

**Issue by matthewrsj** [clearlinux/mixer-tools#337]
```
# mkdir -p /usr/share/mix/rpms

# cp <rpms> /usr/share/mix/rpms

# mixin package add <rpm>
Initializing mix version 22780000 from upstream version 22780
Adding bundle "os-core" from upstream bundles
ERROR: no such package

# mixin package add <rpm>    # trying again...
Initializing mix version 22780000 from upstream version 22780
Bundle "os-core" already in mix; skipping
ERROR: open : no such file or directory
```

An issue seems to be that `mixin/helpers.go` is writing the default configuration using the `UseNewConfig` option, but it is being read everywhere using the old configuration parsing.

**Comment by matthewrsj** [comment]
Some more information, when execution reaches https://github.com/clearlinux/mixer-tools/blob/master/mixin/package.go#L117 `b.Config.Mixer.LocalRPMDir` is an empty string.

**Comment by matthewrsj** [comment]
This was fixed by @rchiossi in #338
